lib/Runtime/Library/ConcatString.cpp

```diff
@@ -99,24 +99,24 @@ namespace Js
         }
 
         ScriptContext * scriptContext = library->GetScriptContext();
-        size_t cbDestString = (charCount + 1) * sizeof(WCHAR);
-        if ((CharCount)cbDestString < charCount) // overflow
+        if (charCount > MaxCharCount) // overflow
```

Comment:
"overflow" comment might be out of date; I think it was referring to arithmetic overflow when computing `(charCount + 1) * sizeof(WCHAR)`